styleguide/components/31-molecules/wizard-steps/_wizard-steps.scss

```diff
@@ -0,0 +1,164 @@
+.wizard-steps {
+
+  ol {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0;
+    counter-reset: item;
+    list-style-type: none;
+
+    li {
+      display: inline-block;
+      margin-bottom: .8rem;
+
+      &:not(:first-child):not(:last-child):not(.active) {
+        @include tablet-and-below {
+          // clip
+          position: absolute;
+          width: 1px;
+          height: 1px;
+          margin: -1px;
+          padding: 0;
+          border: 0;
+          overflow: hidden;
+          // also hide from screenreaders
+          visibility: hidden;
+          clip: rect(0 0 0 0);
```

Comment:
This exact snippet is also used in the `%visually-hidden` helper class .

https://github.com/StadGent/drupal_theme_gent-base/blob/4669e6218a4288b0d15531441bc4af285d82aacc/styleguide/components/11-base/base/_helper-classes.scss#L13-L26

Shouldn't we create a mixin rather than copying the code over and over?

Reply:
There is, but I can't use it inside of the breakpoint mixin.
Sass does not compile.